frontend/src/EditTimetable.tsx

```diff
@@ -1141,7 +1141,7 @@ function EditTimetable() {
           </TooltipProvider>
         </div>
       ) : (
-        <div className="flex text-muted-foreground gap-8 xl:text-xl lg:text-lg md:text-md text-sm flex-col bg-background h-full pt-96 w-full items-center">
+        <div className="flex flex-col text-muted-foreground gap-8 xl:text-xl lg:text-lg md:text-md text-sm bg-background h-full py-80 w-full items-center">
```

Comment:
Might be better to fix the height of the container and then center the spinner in that:
```suggestion
        <div className="flex flex-col text-muted-foreground gap-8 xl:text-xl lg:text-lg md:text-md text-sm bg-background h-[calc(100dvh-5rem)] justify-center w-full items-center">
```